compiler/quilt/tools/command.py

```diff
@@ -1326,17 +1332,85 @@ def iter_filename_map(node):
         # Technically successful, but with nothing to do.
         # package may have no file nodes, or user may have filtered out all applicable targets.
         # -- should we consider it an error and raise?
-        print("No files to export.")
+        if not silent:
```

Comment:
I don't think we're ready to standardize on silent... For now, just print like other commands.

Then in a separate proposal, slack Dima and request permission to add silent= to all python API commands to disable screen/stdout output.

Reply:
Makes sense.